pegasus/sites.v3/code.org/public/css/page/helloworld-styles.scss

```diff
@@ -0,0 +1,76 @@
+@import 'color', 'font', 'typography', 'breakpoints', 'barlow-semi-condensed-font';
+
+// Hero banner
+.hero-banner-basic {
+
+    &.overlay {
+      position: relative;
+      width: 100%;
+      height: 100%;
+  
+      &::before {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        background: #005F6A;
+        z-index: 1;
+      }
+  
+      &::after {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        background-image: url('/images/banners/banner-bg-squiggle-lines.png');
+        background-repeat: repeat;
+        background-size: contain;
+        z-index: 2;
+      }
+    }
+  
+    @media screen and (max-width: $width-sm) {
+      background-size: 50%;
+    }
+  
+    .wrapper {
+      position: relative;
+      z-index: 10;
+    }
+  
+    .text-wrapper {
+      width: 64%;
+  
+      @media screen and (max-width: $width-sm) {
+        width: 100%;
+      }
+    }
+  
+    .media-wrapper {
+      margin-left: 2rem;
+      width: 40%;
+  
+      @media screen and (max-width: $width-sm) {
+        display: none;
+      }
+  
+      img {
+        width: 100%;
+      }
+    }
+  }
+
+  .helloworld_student_projects {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
```

Comment:
Nit: you can add the following classes to this div and remove these `.flex-container.wrap.justify-center`

Reply:
Hi! I tried adding these to the end of lines 38, 39 and 41 and none of those worked. Can you tell me what I'm doing wrong?
<img width="467" alt="Screenshot 2024-08-29 at 9 28 23 AM" src="https://github.com/user-attachments/assets/4bc7b5ec-f5eb-4239-880d-ab95642121d8">

Reply:
jkjk it's taken care of in the react side of the world!

Reply:
Ah ok that makes sense now, and good to know those classes won't work on a placeholder-y div!